stdlib/private/StdlibCollectionUnittest/CheckRangeReplaceableCollectionType.swift

```diff
@@ -12,14 +12,14 @@
 
 import StdlibUnittest
 
-internal enum RangeSelection {
+public enum RangeSelection {
```

Comment:
I'm going to end up using this enum from at least two more test files. Would it be a good idea to move it into a new 'Utilities' style file?